### DISCUSSION TITLE
how do I use `switches` and `options` in the neogit pane

### DISCUSSION THREAD

**Discussion by molleweide** [NeogitOrg/neogit#264] [Q&A] [answered]
Hi guys,

How do I make use of the switches and options when for example one wants to git log.
I am not sure what command to press in order to use these.
``` vim
Switches
 -g Show graph (--graph)
 -c Show graph in color (--color)
 -d Show refnames (--decorate)
 -S Show signatures (--show-signature)
 -u Show diffs (--patch)
 -s Show diffstats (--stat)
 -D Simplify by decoration (--simplify-by-decoration)
 -f Follow renames when showing single-file log (--follow)

Options
 =n Limit number of commits (--max-count=256)
 =f Limit to files (--count=)
 =a Limit to author (--author=)
 =g Search messages (--grep=)

Actions
l Log current  b Log all branches   r Reflog current
o Log other    a Log all references O Reflog other  
h Log HEAD                          H Reflog HEAD   
```

**Comment by TimUntersberger** [comment, marked as answer]
You can either hit the key combination infront of the description (like `-g` for `Show graph`)
or move your cursor ontop of the line containing the switch you want to change and pressing the `tab` key.

Same thing for options.

**Reply by molleweide** [reply]
aaaaah thanks now I understand. with some of the switches the change was so subtle that I just didn't understand if anything was changing or taking effect. thanks for replying. this is just great!!!
